templating/base_render_options_test.go

```diff
@@ -137,6 +137,40 @@ func TestLoadAndRenderSteps(t *testing.T) {
 	}
 }
 
+func TestLoadAndRenderBuildSteps(t *testing.T) {
+	opts := &BaseRenderOptions{
+		ValuesFile: "",
+	}
+	tests := []struct {
+		buildFile string
+		expected  string
+	}{
+		{
+			"testdata/caching/Dockerfile-test",
```

Comment:
May I suggest use a real case for this testcase? In build command, the template string is something like

`--pull --label a=b --no-cache -f Dockerfile --bug-arg c=d -t {{.Run.Registry}} --platform linux/amd64 /workspace`

Reply:
yep, u can create a template using something like:

https://github.com/Azure/acr-builder/blob/master/cmd/acb/commands/build/build.go#L315